packages/expo-store-review/ios/StoreReviewModule.swift

```diff
@@ -0,0 +1,26 @@
+import ExpoModulesCore
+import StoreKit
+
+public class StoreReviewModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoStoreReview")
+
+    AsyncFunction("isAvailableAsync") { () -> Bool in
+      return true
+    }
+
+    AsyncFunction("requestReview") { (promise: Promise) -> Void in
+      if #available(iOS 15, *) {
+        guard let currentScene = UIApplication.shared.connectedScenes.first as? UIWindowScene else {
+          return promise.reject(MissingCurrentWindowSceneException())
+        }
+
+        SKStoreReviewController.requestReview(in: currentScene)
+        promise.resolve(nil)
```

Comment:
If the promise is not in the arguments list, you can just `return nil` which will also resolve the JS promise with `null`. We can go even further and don't return at all 😉 

```suggestion
```